code/game/machinery/door_control.dm

```diff
@@ -138,6 +138,7 @@
 	. = ..()
 	radio_conn = SSradio.add_object(src, BLAST_DOOR_FREQ, RADIO_BLASTDOORS)
 	AddComponent(/datum/component/overlay_manager)
+	return INITIALIZE_HINT_LATELOAD
```

Comment:
Redundant, this proc already returns `INITIALIZE_HINT_LATELOAD` due to `. = ..()` statement.
```suggestion
```